src/pages/index.js

```diff
@@ -108,36 +134,126 @@ function handleModalEscape(evt) {
   }
 }
 
+//Opens modal
 function openModal(modal) {
   modal.classList.add("modal_opened");
   modal.addEventListener("click", handleModalOverlay);
   document.addEventListener("keydown", handleModalEscape);
 }
 
+//Closes modal
 function closeModal(modal) {
   modal.classList.remove("modal_opened");
   modal.removeEventListener("click", handleModalOverlay);
   document.removeEventListener("keydown", handleModalEscape);
 }
 
+//Profile submit handler
 function handleProfileFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = modalNameInput.value;
-  profileDescription.textContent = modalDescriptionInput.value;
-  closeModal(modalEdit);
+
+  const submitBtn = evt.submitter;
+  submitBtn.textContent = "Saving...";
+
+  api
+    .editUserInfo({
+      name: modalNameInput.value,
+      about: modalDescriptionInput.value,
+    })
+    .then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
+      closeModal(modalEdit);
+    })
+    .catch(console.error)
+    .finally(() => {
+      submitBtn.textContent = "Save";
+    });
 }
 
-function handleCardFormSubmit(evt) {
+//Avatar submit handler
+function handleAvatarSubmit(evt) {
   evt.preventDefault();
-  const inputValues = {
-    name: cardCaptionInput.value,
-    link: cardLinkInput.value,
-  };
-  const cardElement = getCardElement(inputValues);
-  cardsList.prepend(cardElement);
-  modalCardForm.reset();
-  disabledButton(cardSubmitBtn, settings);
-  closeModal(modalCard);
+
+  const submitBtn = evt.submitter;
+  submitBtn.textContent = "Saving...";
+
+  api
+    .editAvatarInfo({
+      avatar: avatarLinkInput.value,
+    })
+    .then((data) => {
+      // console.log(data);
+      profileAvatar.src = data.avatar;
+      closeModal(avatarModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+      submitBtn.textContent = "Save";
+    });
+}
+
+//Card submit handler
+function handleCreateCard(evt) {
+  evt.preventDefault();
+
+  const submitBtn = evt.submitter;
+  submitBtn.textContent = "Saving...";
+
+  api
+    .addCards({
+      name: cardCaptionInput.value,
+      link: cardLinkInput.value,
+    })
+    .then((data) => {
+      // getCardElement(data);
+      const cardElement = getCardElement(data);
+      cardsList.prepend(cardElement);
```

Comment:
* You should **clear the inputs** after a successful adding of a new card to let the user add the 2nd one again without having to remove the old data manually.  

`modalCardForm.reset()`